sitewit/services.py

```diff
@@ -65,3 +65,52 @@ def create_account(self, site_id, url, user_name, user_email,
 
         return self.post(
             '/api/account/', data, headers=self._get_auth_header()).json()
+
+    def get_account(self, account_token):
+        """ Get SiteWit account.
+
+        Args:
+            account_token (str): account token.
+
+        Returns:
+            account json:
+            https://sandboxpapi.sitewit.com/Help/Api/GET-api-Account
+        """
+        return self.get(
+            '/api/account/',
+            headers=self._get_auth_header(account_token)).json()
```

Comment:
There is a common pattern `self.METHOD('/api/account/', headers=self._get_auth_header(account_token) ).json()` we can move it to a separate method to reduce code duplication.